packages/tools/migrations-irec/config/demo-config.json

```diff
@@ -89,64 +89,31 @@
-        ["Hydro-electric Head"],
-        ["Hydro-electric Head", "Run-of-river head installation"],
-        ["Hydro-electric Head", "Storage head installation"],
-        ["Hydro-electric Head", "Pure pumped storage head installation"],
-        ["Hydro-electric Head", "Mixed pumped storage head"],
-        ["Marine"],
-        ["Marine", "Tidal"],
-        ["Marine", "Tidal", "Inshore"],
-        ["Marine", "Tidal", "Offshore"],
-        ["Marine", "Wave"],
-        ["Marine", "Wave", "Onshore"],
-        ["Marine", "Wave", "Offshore"],
-        ["Marine", "Currents"],
-        ["Marine", "Pressure"],
-        ["Marine", "Thermal"],
-        ["Solid"],
-        ["Solid", "Muncipal waste"],
-        ["Solid", "Muncipal waste", "Biogenic"],
-        ["Solid", "Industrial and commercial waste"],
-        ["Solid", "Industrial and commercial waste", "Biogenic"],
-        ["Solid", "Wood"],
-        ["Solid", "Wood", "Forestry products"],
-        ["Solid", "Wood", "Forestry by-products & waste"],
-        ["Solid", "Animal fats"],
-        ["Solid", "Biomass from agriculture"],
-        ["Solid", "Biomass from agriculture", "Agricultural products"],
-        ["Solid", "Biomass from agriculture", "Agricultural by-products & waste"],
-        ["Liquid"],
-        ["Liquid", "Municipal biodegradable waste"],
-        ["Liquid", "Black liquor"],
-        ["Liquid", "Pure plant oil"],
-        ["Liquid", "Waste plant oil"],
-        ["Liquid", "Refined vegetable oil"],
-        ["Liquid", "Refined vegetable oil", "Biodiesel (mono-alkyl ester)"],
-        ["Liquid", "Refined vegetable oil", "Biogasoline (C6-C12 hydrocarbon)"],
-        ["Gaseous"],
-        ["Gaseous", "Landfill gas"],
-        ["Gaseous", "Sewage gas"],
-        ["Gaseous", "Agricultural gas"],
-        ["Gaseous", "Agricultural gas", "Animal manure"],
-        ["Gaseous", "Agricultural gas", "Energy crops"],
-        ["Gaseous", "Gas from organic waste digestion"],
-        ["Gaseous", "Process gas"],
-        ["Gaseous", "Process gas", "Biogenic"],
-        ["Thermal"],
-        ["Thermal", "Internal combustion engine"],
-        ["Thermal", "Internal combustion engine", "Non CHP"],
-        ["Thermal", "Internal combustion engine", "CHP"],
-        ["Thermal", "Steam turbine with condensation turbine"],
-        ["Thermal", "Steam turbine with condensation turbine", "Non CHP"]
+        ["ES100", "TC110"],
+        ["ES100", "TC120"],
+        ["ES100", "TC130"],
+        ["ES100", "TC140"],
+        ["ES100", "TC150"],
+
+        ["ES200", "TC210"],
+        ["ES200", "TC220"],
+
+        ["ES300", "TC310"],
+        ["ES300", "TC320"],
+        ["ES300", "TC330"],
+
+        ["ES410", "TC410"],
+        ["ES410", "TC411"],
+
+        ["ES420", "TC421"],
+        ["ES420", "TC422"],
+        ["ES420", "TC423"],
+        ["ES420", "TC424"],
+
+        ["ES430", "TC431"],
+        ["ES430", "TC432"],
+
+        ["ES440", "TC441"],
+        ["ES440", "TC442"],
```

Comment:
Since IREC does not have any mapping between fuel and device codes, I decided to put here just a small subset of available types. Otherwise, I would have to put here a multiplication of 38 device types and 31 fuel types.

Also Dan said that they are going to implement a new system behind it, so we will have to update this list soon.

Reply:
Use `name` instead of `code`.